addon-manager/tests/pom.xml

```diff
@@ -23,6 +23,12 @@
       </dependency>
       <dependency>
          <groupId>org.jboss.forge.addon</groupId>
+         <artifactId>ui-impl</artifactId>
```

Comment:
What I meant is that it feels wrong to expose impl from another modules inside the test module for addon-manager.